src/main/scala/objsets/TweetSet.scala

```diff
@@ -54,7 +56,9 @@ abstract class TweetSet extends TweetSetInterface {
    * Question: Should we implment this method here, or should it remain abstract
    * and be implemented in the subclasses?
    */
-  def union(that: TweetSet): TweetSet = ???
+  def union(that: TweetSet): TweetSet = {
+    that.filterAcc(tweet => true, this)
```

Comment:
```suggestion
    that.filterAcc(_ => true, this)
```
If you aren't using a variable, it's a good practice to let sbt (and the reader) know about it in advanced